### PR TITLE
Read the docs

### PR DESCRIPTION
**Pull request by K-Gazdikova**
I have created our ticketutil project on readthedocs which makes referring our documentation much more easier.
The link for the documentation can be found at : http://ticketutil.readthedocs.io/en/latest/index.html
Please review and let me know your feedback.

Thanks,

### CODE REVIEW

File: read-the-docs/source/Installation.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Installation
+============
+
+* Install ticketutil with,
```
Comment (dmranck) [inline read-the-docs/source/Installation.rst]: How about:
Install ticketutil with:

File: read-the-docs/source/Installation.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    pip install ticketutil
+
+* ticketutil is compatible with Python 2.7, 3.3, 3.4, 3.5, and 3.6.
+
+.. note::
+
+    For Python 2.6 and lower, an additional package, importlib, may need to be installed.
+
+* If not installing with pip, a short list of packages defined in the requirements.txt file need to be installed. To install the required packages with,
```
Comment (dmranck) [inline read-the-docs/source/Installation.rst]: How about:
To install the required packages, type:

File: read-the-docs/source/Usage.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
++ Create a ticket with the ``create()`` method. This sets the ``ticket_id``
+  instance variable, allowing you to perform more tasks on the ticket.
+
++ Add comments, edit ticket fields, add watchers, change the ticket
+  status, etc on the ticket.
+
++ Close ticket Requests session with ``close_requests_session()``.
+
++ To work on existing tickets, you can also pass in a fourth parameter
```
Comment (dmranck) [inline read-the-docs/source/Usage.rst]: Lines 27 - 38 describe working with existing tickets. This should be moved under the "Work with an existing ticket" section.

File: read-the-docs/source/Usage.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+To run unit tests in Bash terminal use this command:
+
+.. code-block:: python
+
+    python3 -m unittest discover ./tests/
+
+.. seealso::
+
+    See the docstrings in the code or the tool-specific files in the docs
```
Comment (dmranck) [inline read-the-docs/source/Usage.rst]: This sentence doesn't make sense anymore for readthedocs. What about:
See the docstrings in the code or the tool-specific sections in the documentation for more information on supported methods and examples.

File: read-the-docs/source/ServiceNow.rst [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+non-existent in another table.
+
+Custom field names and values can be passed in as keyword arguments when
+creating and editing tickets. These are processed by the ServiceNow REST
+API right afterwards. You can also use
+`get\_ticket\_content <#content>`__ to get ticket content including
+preset field names.
+
+For more information about REST API and fields see ServiceNow's Wiki: -
+`REST API <http://wiki.servicenow.com/index.php?title=REST_API>`__ -
```
Comment (dmranck) [inline read-the-docs/source/ServiceNow.rst]: These three links got jumbled together on readthedocs. They should be in a bulleted list as they are in: https://github.com/dmranck/ticketutil/blob/master/docs/servicenow.md

File: read-the-docs/source/Installation.rst (hunk 1 above)
Comment (K-Gazdikova) [inline read-the-docs/source/Installation.rst]: done !

File: read-the-docs/source/Installation.rst (hunk 2 above)
Comment (K-Gazdikova) [inline read-the-docs/source/Installation.rst]: done !

File: read-the-docs/source/Usage.rst (hunk 3 above)
Comment (K-Gazdikova) [inline read-the-docs/source/Usage.rst]: done !

File: read-the-docs/source/Usage.rst (hunk 4 above)
Comment (K-Gazdikova) [inline read-the-docs/source/Usage.rst]: done!

File: read-the-docs/source/ServiceNow.rst (hunk 5 above)
Comment (K-Gazdikova) [inline read-the-docs/source/ServiceNow.rst]: done!

**Comment by K-Gazdikova** [comment]
Hi Danny, 
As per our chat conversation adding the below modification requests,
1) Update readthedocs config file to refer our master branch for readthedocs branch.
2) Update  index.html file at the top (the code example) with the ticket and t changes.
3)  Add the return statements section under Usage file.

File: read-the-docs/source/Usage.rst [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+Running unit tests
+------------------
+
+To run unit tests in Bash terminal use this command:
+
+.. code-block:: python
+
+    python3 -m unittest discover ./tests/
+
+.. seealso::
```
Comment (dmranck) [inline read-the-docs/source/Usage.rst]: This seealso section should move to the end of the "Work with an existing ticket" section.

File: read-the-docs/source/Usage.rst [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+    See the docstrings in the code or the tool-specific sections in the documentation for more information on
+    supported methods and examples.
+
+.. note::
```
Comment (dmranck) [inline read-the-docs/source/Usage.rst]: Let's move this note to the end of the "Create a ticket object" section. Logging is important and this note is currently kind of buried in this page.

File: read-the-docs/source/Usage.rst (hunk 6 above)
Comment (K-Gazdikova) [inline read-the-docs/source/Usage.rst]: done!

File: read-the-docs/source/Usage.rst (hunk 7 above)
Comment (K-Gazdikova) [inline read-the-docs/source/Usage.rst]: done

**Pull request merged**